lib/LaTeXML/Package/psfrag.sty.ltxml

```diff
@@ -33,6 +33,9 @@ ProcessOptions();
 
 sub save_psfrag {
   my (@stuff) = @_;
+  # avoid saving if nothing given.
+  @stuff = @stuff && Expand(@stuff);
```

Comment:
I guess `Expand` is ok here? The stuff gets digested later.  But I suppose you're trying to avoid having pointless fragments so as to avoid unnecessary LaTeXImages, right?

Reply:
My hope is that the Expand helps a more often than it breaks here.

The reason I added it was to expand custom macros that weren't otherwise available "in vacuum" for the call to `latex ltximg.tex`. From what I remember, the example I had was a macro introduced via `\DeclareMathSymbol` inside a psfrag argument, which was copied as-is and was unknown at the time latex was called.

Reply:
But all user macros should have been digested out later on (which also expands), before it ends up in the TeX attribute and ltximg.  Expand can be dangerous, because it defeats the usual interleaving of expansion/digestion.

Reply:
I guess I can remove it, not much gained or lost either way.